lib/recipeData.tsx

```diff
@@ -0,0 +1,131 @@
+import { LoginType } from './types';
+import LoginWithSMS from '../components/SMSPasscodes/LoginWithSMS';
+import LoginWithEmailWebAuthn from '../components/EmailWebAuthn/LoginWithEmail';
+import LoginWithMagicLinks from '../components/LoginWithReactSDK';
+import LoginProducts from './loginProduct';
+
+export const Recipes: Record<string, LoginType> = {
+  REACT: {
+    id: 'ui-js-sdk',
+    title: 'Pre-built UI + JavaScript SDK',
+    details:
+      'Use our pre-built UI component and JavaScript SDK to get started with Stytch as quickly as possible. The pre-built UI provides a beautiful and customizable login form to make sure your brand stays front and center and the SDK handles everything else for you.',
+    description:
+      'In this recipe we demonstrate a login flow that includes Email magic links and several OAuth options and Google One Tap.',
+    instructions: `To the right you'll see our pre-built login form with several OAuth providers and Email magic links. Below you can see the configuration and customization parameters used to create the login form.`,
+    component: <LoginWithMagicLinks />,
+    products: [LoginProducts.EML, LoginProducts.OAUTH],
+    code: `
+      const sdkStyle: StyleConfig = {
+        fontFamily: '"Helvetica New", Helvetica, sans-serif',
+        primaryColor: '#19303d',
+        primaryTextColor: '#090909',
+      };
+      
+      const magicLinksView = {
+        products: [SDKProductTypes.oauth, SDKProductTypes.emailMagicLinks],
+        emailMagicLinksOptions: {
+          loginRedirectURL: REDIRECT_URL_BASE + '/authenticate?type=em',
+          loginExpirationMinutes: 30,
+          signupRedirectURL: REDIRECT_URL_BASE + '/authenticate?type=em',
+          signupExpirationMinutes: 30,
+          createUserAsPending: false,
+        },
+        oauthOptions: {
+          providers: [
+            { type: OAuthProvidersTypes.Google, one_tap: true, position: 'embedded' },
+            { type: OAuthProvidersTypes.Apple },
+            { type: OAuthProvidersTypes.Microsoft },
+            { type: OAuthProvidersTypes.Facebook },
+            { type: OAuthProvidersTypes.Github },
```

Comment:
```suggestion
            { type: OAuthProvidersTypes.Github },
            { type: OAuthProvidersTypes.GitLab },
```
We missed hard coding that addition here.

Reply:
Will add!